test/bdd/pkg/kms/zcap.go

```diff
@@ -39,133 +27,32 @@ type signer interface {
 	Alg() string
 }
 
-type authzKMSSigner struct {
+type zCapAuthUserSigner struct {
```

Comment:
Should be `zcapAuthUserSigner` not `zCapAuthUserSigner`

Reply:
(`ZCAP` is an acronym, so should be in all caps, except when at the beginning of an unexported name, in which case it's all lowercase)

Reply:
fixed